apps/src/lib/ui/accounts/DeleteAccountDialog.story.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import DeleteAccountDialog from './DeleteAccountDialog';
+
+const PASSWORD = 'MY_PASSWORD';
+const DELETE_VERIFICATION = 'DELETE MY ACCOUNT';
+
+export default storybook => {
+  storybook
+    .storiesOf('Dialogs/DeleteAccountDialog', module)
+    .addStoryTable([
+      {
+        name: 'Delete Student Account',
+        description: 'Warning message for student account deletion',
+        story: () => (
+          <DeleteAccountDialog
+            isOpen={true}
```

Comment:
a lot of the props for this story and the next one are repeated. i would pull them out into a `DEFAULT_PROPS` object to make duplicating these stories easier in the future. there's an example [here](https://github.com/code-dot-org/code-dot-org/blob/4ba87b8b8553852f08f20954c701c45161e5ad71/apps/src/templates/manageStudents/MoveStudents.story.jsx#L35-L44)

Reply:
Added default_props object.  Will create another PR 2 additional stories.